From e764765cc9ca9ad5965c72829ceb468a4ff58c66 Mon Sep 17 00:00:00 2001
From: Valentina Fernandez <valentina.fernandezalanis@microchip.com>
Date: Wed, 16 Nov 2022 12:28:21 +0000
Subject: [PATCH] fix(hss-payload-generator): add support for additional host
 include files

Update the hss-payload-generator makefile to allow passing additional
include paths as an argument when using the make command

Signed-off-by: Valentina Fernandez <valentina.fernandezalanis@microchip.com>
---
 tools/hss-payload-generator/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/hss-payload-generator/Makefile b/tools/hss-payload-generator/Makefile
index 74cac4b..6e1efe5 100644
--- a/tools/hss-payload-generator/Makefile
+++ b/tools/hss-payload-generator/Makefile
@@ -54,6 +54,7 @@ CFLAGS= -g3 -ggdb -std=gnu11 -O3 \
 INCLUDES=\
 	-I. \
 	-I../../include \
+	$(HOST_INCLUDES)
 
 LIBS=\
 	-lyaml \
-- 
2.25.1

